tests/unit/testUncaught.js

```diff
@@ -50,6 +52,13 @@ test('Uncaught handler setup', function (t) {
 });
 
 test('Test uncaught shutdown behavior', function (t) {
+  if (!isString(process.env.GOOGLE_APPLICATION_CREDENTIALS)
```

Comment:
@matthewloring Wouldn't this then cause the ci to always fail on pr's arising externally? Istanbul does log skipped tests as pending, not sure if that is enough in this case though.